software/base/src/main/java/brooklyn/entity/software/StaticSensor.java

```diff
@@ -36,9 +43,17 @@ public StaticSensor(ConfigBag params) {
         value = params.get(STATIC_VALUE);
     }
 
+    @SuppressWarnings("unchecked")
     @Override
     public void apply(EntityLocal entity) {
         super.apply(entity);
-        entity.setAttribute(sensor, (T) TypeCoercions.coerce(value, sensor.getType()));
+        
+        Maybe<T> v = Tasks.resolving(value).as((Class<T>)sensor.getType()).timeout(Duration.millis(200)).getMaybe();
```

Comment:
100% agree re arbitrary numbers in code.  i've replace them with defined constants whose name and javadoc give a bit of reasoning behind the arbitrariness.  lemme know if you think it's an improvement @neykov .